desktop.css

```diff
@@ -0,0 +1,296 @@
+    left -10px top 1150px,
+    right 0 top 1600px,
+    left -100px top 2500px,
+    right -10px top 2450px;
+}
+
+.main-page-header {
+  display: none;
+}
+
+.header-hidden {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: #ffff;
+  padding: 0 142px;
+  margin-bottom: 0;
+}
+
+.nav-bar {
+  display: flex;
+  gap: 20px;
+}
+
+.nav-links {
+  display: flex;
+  gap: 20px;
+  font-size: 0.93rem;
+}
+
+.message {
+  display: flex;
+  align-items: center;
+  padding-bottom: 4px;
+  align-content: center;
+}
+
+.work-element-hidden {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: #ffff;
+  margin-bottom: 0;
+  padding: 0 40px;
+}
+
+.logo {
+  padding: 10px 0;
+}
+
+.open-menu {
+  padding: 10px 0;
+}
+
+.headline {
+  padding: 130px 142px 114px;
+  gap: 88px;
+}
+
+.name {
+  font-size: 5.62rem;
+  text-align: center;
+  line-height: 105px;
+}
+
+.occupation {
+  font-size: 2rem;
+  margin-bottom: 0;
+  padding: 0;
+  text-align: center;
+  line-height: 47px;
+}
+
+.about-text {
+  margin: 0;
+  padding: 0 10px 45px;
+  text-align: justify;
+  text-align-last: center;
+  line-height: 32px;
+  left: 0.5px;
+  width: 100%;
+}
+
+.about-text-links {
+  padding: 0 354px;
+}
+
+.about-links {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: flex-start;
+  padding-top: 50px;
+  gap: 23px;
+}
+
+.about-l-section {
+  width: 30%;
+}
+
+.about-l-header {
+  font-size: 1.12rem;
+  font-weight: 700;
+}
+
+.work-element {
+  display: none;
+}
+
+.social-media {
+  display: none;
+}
+
+.social-media-hidden {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  gap: 8px;
+  padding: 0;
+  margin-top: 0;
+}
+
+.recent-works {
+  display: grid;
+  padding: 24px 142px 114px;
+  gap: 88px;
+  text-align: left;
+  justify-content: center;
+}
+
+.recent-works-header-hidden {
+  display: none;
+}
+
+.recent-works-header {
+  display: block;
+  color: #9490ff;
+  max-width: 100%;
+  font-weight: 700;
+  font-size: 4rem;
+  line-height: 114px;
+  padding-bottom: 38px;
+  border-bottom: 1px solid #646369;
+  margin-top: 65px;
+}
+
+.work-card {
+  display: grid;
+  grid-template-columns: 60% 35%;
+  gap: 20px;
+}
+
+.left-block {
+  display: grid;
+  grid-template-columns: 35% 60%;
+  gap: 20px;
+}
+
+.left-side {
+  order: 1;
+  width: 100%;
+}
+
+.image-placeholder {
+  width: 100%;
+  height: auto;
+}
+
+.work-content {
+  display: flex;
+  flex-direction: column;
+  gap: 28px;
+  align-items: flex-start;
+}
+
+.work-header {
+  font-size: 2.5rem;
+  font-weight: 700;
+  margin-bottom: 0;
+}
+
+.work-p {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.tags-section {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 0;
+  margin: 0;
+}
+
+.tag {
+  flex: content;
+  padding: 0 18px;
+  border-right: 1px solid #646369;
+}
+
+.ruby-tag {
+  border-right: none;
+}
+
+.work-card-button:enabled {
+  margin-top: 0;
+}
```

Comment:
I apologize if previous reviewers didn't mention this, but you should change the `:enabled` pseudo-class to `:active`, that way when the user clicks on a button the style changes only while the button is being clicked and it doesn't persist after.